### ISSUE TITLE
Add RMS and RMS Peak to Loudness Analyze table

### ISSUE THREAD

**Issue by MrBeignets** [reaper-oss/sws#867]
Hi,
Could it be possible to have "RMS" and "RMS Peak" as columns in the Loudness window, following the settings in SWS RMS Options ?

It's really useful as the analyze can be them exported in a spreadsheet.

Thanks!

**Comment by teeramusic** [comment]
Yes please! Also option to show peak without calculating true peak
